pkg/kubelet/cm/dra/plugin/client.go

```diff
@@ -30,7 +30,14 @@ import (
 	drapb "k8s.io/kubelet/pkg/apis/dra/v1alpha3"
 )
 
-const PluginClientTimeout = 45 * time.Second
+var (
+	PluginClientTimeout = 45 * time.Second
```

Comment:
Now `PluginClientTimeout` becomes a global variable, which is not good from my point of view. Can we instead add `timeout` field to the `plugin.go::plugin` structure instead and set it when plugin is registered? We can pass desired timeout value to the `plugin.go::RegisterPlugin` from the test.

Reply:
+1 to this.

Reply:
/hold
until this is addressed

Reply:
Looks better, I initially wanted as few invasive changes to the code as possible :)

Reply:
/hold cancel

Based on https://github.com/kubernetes/kubernetes/pull/123720#issuecomment-2004119139